### PR TITLE
Lets AI eyes move diagonally

### PR DESCRIPTION
**Pull request by DamianX**
Doesn't _really_ matter, but why not?

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/login.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/mob/living/silicon/ai/Login()	//ThisIsDumb(TM) TODO: tidy this up �_� ~Carn
+/mob/living/silicon/ai/Login()	//ThisIsDumb(TM) TODO: tidy this up �_� ~Carn
```
Comment (SonixApache) [inline code/modules/mob/living/silicon/ai/login.dm]: ¬_¬

Comment (DamianX) [reply code/modules/mob/living/silicon/ai/login.dm]: �_�

Comment (SonixApache) [reply code/modules/mob/living/silicon/ai/login.dm]: fix iiiit

**Comment by ghost** [comment]
What the fuck you're buffing AI? I need an angery emoji for this!

**Comment by SonixApache** [comment]
😡 *carbons*

**Pull request merged**